docs/src/components/Markdown.js

```diff
@@ -28,27 +29,65 @@ const stripIndent = (str: string): string => {
   return str.replace(re, '');
 };
 
-export default function Markdown({ text }: Props): Node {
-  const renderer = new Renderer();
+const makePullRequestLink = listitem => {
+  const pullRequestNumRegex = /\(#\d*\)/g;
+  let pullRequestNum = listitem.match(pullRequestNumRegex);
+  if (pullRequestNum && pullRequestNum.length > 0) {
+    pullRequestNum = pullRequestNum[0].replace(/\D/g, '');
+    const pullRequestURL = `https://github.com/pinterest/gestalt/pull/${pullRequestNum}`;
+    const pullRequestAnchor = `(<a href=${pullRequestURL}>#${pullRequestNum}</a>)`;
+    const listItemWithLink = listitem.replace(
+      pullRequestNumRegex,
+      pullRequestAnchor
+    );
+    return listItemWithLink;
+  }
+  return listitem;
+};
+
+const formatCmpName = listitem => {
+  const cmpNamesToIgnore = ['doc', 'docs', 'internal', 'codemod'];
```

Comment:
@ayeshakmaz this doesn't always seem to work, especially when there is more than 1 component.

`ActivationCard/Callout` links to https://gestalt.netlify.app/ActivationCard/Callout - which is not a valid page.

Might be worthwhile to dynamically pull in all the Component Pages + replace those

![image](https://user-images.githubusercontent.com/127199/96320570-96edaf80-0fc7-11eb-98b0-69608545c89a.png)

Reply:
Yeah this one is funky - it still does work, in the sense that it will just bring you to the first cmp's page (ActivationCard in that example ^) but with a wonky URL. I can make an update to look for slashes, and link each cmp individually, what do you think?